templates/definition/vehicle/teslalogger.yaml

```diff
@@ -0,0 +1,83 @@
+template: teslalogger
+products:
+  - description:
+      generic: TeslaLogger
+group: generic
```

Comment:
@naltatis generic passt hier nicht? Wie wollen wir diese "Adapter" positionieren?

Reply:
Es sind ja einige von solchen Adaptern in generic
Evtl. eine Gruppe: Logger bzw. Virtual Vehicles